payments/trustly/redirect.md

```diff
@@ -0,0 +1,238 @@
+|                  | └─➔&nbsp;`lastName`               | `string`      | Prefilled value to put in the last name text box.                                                                                                                                                                                                                                                                     |
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": {
+        "id": "/psp/trustly/payments/{{ page.payment_id }}",
+        "number": 99590008046,
+        "created": "2020-05-26T12:31:19.3106483Z",
+        "updated": "2020-05-26T12:31:19.4513673Z",
+        "instrument": "Trustly",
+        "operation": "Purchase",
+        "intent": "Sale",
+        "state": "Ready",
+        "currency": "SEK",
+        "prices": {
+            "id": "/psp/trustly/payments/{{ page.payment_id }}/prices"
+        },
+        "amount": 0,
+        "description": "Test Purchase",
+        "payerReference": "SomeReference",
+        "initiatingSystemUserAgent": "PostmanRuntime/7.25.0",
+        "userAgent": "Mozilla/5.0...",
+        "language": "sv-SE",
+        "urls": {
+            "id": "/psp/trustly/payments/{{ page.payment_id }}/urls"
+        },
+        "payeeInfo": {
+            "id": "/psp/trustly/payments/{{ page.payment_id }}/payeeinfo"
+        },
+        "metadata": {
+            "id": "/psp/trustly/payments/{{ page.payment_id }}/metadata"
+        }
+    },
+    "operations": [
+        {
+            "method": "PATCH",
+            "href": "{{ page.api_url }}/psp/trustly/payments/{{ page.payment_id }}",
+            "rel": "update-payment-abort"
+        },
+        {
+            "method": "POST",
+            "href": "{{ page.api_url }}/psp/trustly/payments/{{ page.payment_id }}/sales",
+            "rel": "create-sale"
+        },
+        {
+            "method": "GET",
+            "href": "https://ecom.externalintegration.payex.com/trustly/payments/sales/8f3ba6c8f4e3f6125ae6c18bec15c612747cf2c35dc5cac35d4bebc10cf7317e",
+            "rel": "redirect-sale"
+        }
+    ]
+}
+```
+
+## Trustly flow
+
+The sequence diagram below shows the two requests you have to send to Swedbank
+Pay to make a purchase.
+The diagram also shows the process of a complete purchase in high level.
+
+```mermaid
+sequenceDiagram
+    Consumer->>Merchant: Start purchase
+    activate Merchant
+    note left of Merchant: First API request
+    Merchant->>-Swedbank Pay: POST <Trustly Payment> (operation=Purchase)
+    activate Swedbank Pay
+    Swedbank Pay-->>-Merchant: payment resource
```

Comment:
Use code-like names for the identifiers (not textual descriptions) within the diagram and define an alias for it with `participant SwedbankPay as Swedbank Pay` at the top instead.

```suggestion
    SwedbankPay-->>-Merchant: payment resource
```